catalog_impls/src/volatile.rs

```diff
@@ -17,51 +16,82 @@ use catalog::{
     schema::{
         self, CatalogMismatch, CloseOptions, CloseTable, CloseTableRequest, CreateOptions,
         CreateTable, CreateTableRequest, DropOptions, DropTable, DropTableRequest, NameRef,
-        OpenOptions, OpenTable, OpenTableRequest, Schema, SchemaMismatch, SchemaRef,
+        OpenOptions, OpenTable, OpenTableRequest, Schema, SchemaMismatch, SchemaName, SchemaRef,
     },
     Catalog, CatalogRef,
 };
-use log::info;
+use log::{debug, info};
 use snafu::{ensure, ResultExt};
 use table_engine::table::{SchemaId, TableId, TableRef};
 use tokio::sync::Mutex;
 
 #[async_trait]
-pub trait SchemaIdGenerator {
+pub trait SchemaIdAlloc: Send + Sync {
```

Comment:
Should we move these common traits (`SchemaIdAlloc` and `TableIdAlloc`) to `catalog_impl/lib.rs`?